docs/application/dotnet/get-started/overview.md

```diff
@@ -1,15 +1,127 @@
+<style>
+#main:before, #main:after {
+    content: "";
+    display: table;
+}
 
--   [Mobile .NET application](mobile/first-app.md)
+.docs-ui-started [class^="docs-ui-"] {
+    width: 33%; 
+    height: auto;
+    padding: 40px 0;
+    text-align: center;
+    border: 0 none;
+    border-top: 1px solid #dadada;
+    border-bottom: 1px solid #dadada;
+    box-sizing: border-box;
+    position: relative;
+    float: left;
+    margin: 0 auto 20px;
+}
 
-    A mobile application is targeted to run on a smart phone. The applications can take advantage of a wide variety of functionalities available on smart phones.
+.docs-ui-started [class^="docs-ui-"]>span {
+    display: block;
+    color: #333;
+    line-height: 29px;
+    position: relative;
+	
+}
 
--   [Wearable .NET application](wearable/first-app.md)
+@media (max-width: 768px)
+.docs-ui-started .docs-ui-wearable:before, .docs-ui-started .docs-ui-tv:before, .docs-ui-started .docs-ui-mobile:before {
+    height: 85px;
+    margin: 0 auto 25px;
+    align: center;
+    background-position: 0 6px;
+}
+.docs-ui-started .docs-ui-wearable:before {
+    content: "";
+    display: block;
+    margin: auto;
+    position: relative;
+    width: 70px;
+    height: 90px;
+    background: url(./media/wearable_getstarted.png) no-repeat center top;
+    background-position: 0 0 !important;
+}
+.docs-ui-started .docs-ui-wearable {
+    width: 33%;
+    padding-left: 50;
+    /* border-right: 1px solid #d1d1d1; */
+}
 
-    A wearable application is targeted to run on a smart watch. The applications can be optimized for the smaller screen.
+.docs-ui-started .docs-ui-tv:before {
+    content: "";
+    margin: auto;
+    position: relative;
+    display: block;
+    width: 85px;
+    height: 90px;
+    background: url(./media/tv_getstarted.png) no-repeat center top;
+    background-position: 0 0 !important;
+}
+.docs-ui-started .docs-ui-tv {
+    width: 33%;
+    padding-left: 50;
+}
+
+.docs-ui-started .docs-ui-mobile:before {
+    content: "";
+    margin: auto;
+    position: relative;
+    display: block;
+    width: 70px;
+    height: 90px;
+    background: url(./media/mobile_getstarted.png) no-repeat center top;
+    background-position: 0 0 !important;	
+}
+.docs-ui-started.docs-ui-mobile {
+    width: 33%;
+    padding-left: 50px;	
+}
+
+div {
+    display: block;
+}
+
+a.docs-btn-more {
+    display: inline-block;
+    font-size: 14px;
+    color: #008aee;
+}
+</style>
+
+<section id ="main">
+Tizen .Net applications are developed in C# with numerous advantages such as better security features, debugging services, better versioning, structured exception handling, cross-platform development, and so on. 
+
+Visual Studio and Visual Studio Tools for Tizen are used to develop Tizen .Net applications. To create your first Tizen .NET application, the following profiles are available:
```

Comment:
For better clarity can we rephrase as:

Visual Studio IDE with [Visual Studio Tools for Tizen](https://marketplace.visualstudio.com/items?itemName=tizen.VisualStudioToolsforTizen) is used to develop Tizen .Net applications.